pycookiecheat/pycookiecheat.py

```diff
@@ -64,23 +64,30 @@ def chrome_decrypt(encrypted_value: bytes, key: bytes, init_vector: bytes) \
     return clean(decrypted)
 
 
-def get_osx_config() -> dict:
-    """Get settings for getting Chrome cookies on OSX.
+def get_osx_config(browser_name: str = None) -> dict:
```

Comment:
Can we change the default argument to `Chrome`? -- this should preserve backwards compatibility. You may known more than me on this, but I don't think the `None` and `if x == None` pattern is critical here because strings are immutable -- I generally only use that pattern if I want to have the default argument be a mutable structure.